src/docker.go

```
@@ -1,676 +1,676 @@
+			// 重写WWW-Authenticate头中的realm URL
+			if key == "Www-Authenticate" {
+				// 支持多Registry的URL重写
+				value = rewriteAuthHeader(value, proxyHost)
+			}
+			c.Header(key, value)
+		}
+	}
+
+	// 返回响应
+	c.Status(resp.StatusCode)
+	io.Copy(c.Writer, resp.Body)
```

golangci-lint (1.64.8)

Error return value of `io.Copy` is not checked

(errcheck)